pkg/tests/tasks/migration/migration_test.go

```diff
@@ -90,45 +114,158 @@ spec:
 		oc.DefaultOC.WaitFor(t, smcp.Namespace, "ServiceMeshMemberRoll", "default", `jsonpath='{.status.configuredMembers[?(@=="bookinfo")]}'`)
 
 		t.LogStep("Create 3.0 controlplane and IstioCNI")
-		istioTempl := `apiVersion: sailoperator.io/v1alpha1
-kind: Istio
+		setupIstio(t, istio)
+
+		t.LogStep("Install bookinfo and bookinfo gateway")
+		app.InstallAndWaitReady(t, app.Bookinfo(ns.Bookinfo))
```

Comment:
I'm not sure if strict MTLS is applied here. We probably should apply it to be sure the unmatch traffic is rejected.